pt/lessons/specifics/eex.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: page
+title: Elixir embutido (EEx)
+category: specifics
+order: 3
+lang: pt
+---
+
+Do mesmo jeito que Ruby possui ERB e Java JSPs, Elixir tem EEx ou *Embedded Elixir (Elixir embutido)*. Com EEx podemos embutir e avaliar código Elixir dentro das *strings*.
+
+## Table of Contents
+
+- [API](#API)
+	- [Avaliação](#avaliacao)
+	- [Definições](#definicoes)
+	- [Compilação](#compilacao)
+- [Etiquetas](#etiquetas)
+- [Motor](#motor)
+
+## API
+
+A API EEX suporta trabalhar com cadeias de caracteres e ficheiros directamente. A API está dividida em três componentes principais: avaliação simples, definicações de funções, e complilação para AST.
+
+### Avaliação
+
+Usando `eval_string` e `eval_file` podemos realizar uma simples avaliação sobre uma string ou conteúdos de um ficheiro. Este é a API mais simples mas lento uma vez que o código é interpretado e não compilado.
+
+```elixir
+iex> EEx.eval_string "Hi, <%= name %>", [name: "Sean"]
+"Hi, Sean"
+```
+
+### Definições
+
+A mais rápida e preferida forma de usar o EEx é embutir nosso template dentro de um módulo assim ele pode ser compilado. Para isso precisamos nosso template no momento da compilação e das macros `function_from_string` e `function_from_file`.
```

Comment:
Replace `Para isso precisamos nosso template no momento da compilação e das macros` for
`Para isso precisamos do nosso template no momento da compilação e dos macros`